index.qmd

```diff
@@ -38,3 +38,5 @@ Some text.
 ## Another
 
 More text.
+
+![This is a caption](images/seed-case.png){width="300"}
```

Comment:
Could this maybe be done with the seedcase logo instead of adding this .png?
It's already in the repo here: `seedcase-theme/_extensions/seedcase-theme/logo.png`

Reply:
Yeah, I even had that as a first pass, but wasn't sure if it was okay to reference files inside that folder from outside of it

Reply:
yes that works ☺️ this text is only to make it so we can test out changes to the theme